From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Wed, 29 May 2019 10:58:58 +0200
Subject: [PATCH] Fix --bls-directory option comment in grub2-switch-to-blscfg
 man page

The default directory for BLS fragments is always /boot/loader/entries for
both EFI and non-EFI. Fix the man page accordingly to document this.

Resolves: rhbz#1714835

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 util/grub-switch-to-blscfg.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub-switch-to-blscfg.8 b/util/grub-switch-to-blscfg.8
index 7d99de2d2f9..9a886282976 100644
--- a/util/grub-switch-to-blscfg.8
+++ b/util/grub-switch-to-blscfg.8
@@ -23,7 +23,7 @@ The defaults file for grub-mkconfig.  The default value is \fI/etc/default/grub\
 
 .TP
 --bls-directory=\fIDIR\fR
-Create BootLoaderSpec fragments in \fIDIR\fR.  The default value is \fI/boot/loader/entries\fR on BIOS machines and \fI/boot/efi/EFI/\fBVENDOR\fI/loader/entries\fR on UEFI machines.
+Create BootLoaderSpec fragments in \fIDIR\fR.  The default value is \fI/boot/loader/entries\fR.
 
 .TP
 --backup-suffix=\fSUFFIX\fR
